specification/app/Microsoft.App.DynamicSessions/tspconfig.yaml

```diff
@@ -13,7 +13,6 @@ options:
     output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/DynamicSessions.json"
   "@azure-tools/typespec-python":
     package-dir: "microsoft-app-dynamicsessions"
-    package-name: "{package-dir}"
```

Comment:
removed `package-name` here since the typespec namespace is `Microsoft.App.DynamicSessions`, so no need